discretize/utils/mesh_utils.py

```diff
@@ -693,8 +710,20 @@ def refine_tree_xyz(
     octree_levels = np.asarray(octree_levels)
     octree_levels_padding = np.asarray(octree_levels_padding)
 
+    # levels = mesh.max_level - np.arange(len(octree_levels))
+    # non_zeros = octree_levels != 0
+    # levels = levels[non_zeros]
+
     # Trigger different refine methods
     if method.lower() == "radial":
+        # padding = octree_levels[non_zeros]
+        # mesh.refine_points(xyz, levels, padding, finalize=finalize,)
+        warnings.warn(
+            "The radial option is deprecated as of `0.9.0` please update your code to "
+            "use the `TreeMesh.refine_points` functionality. It will be removed in a "
+            "future version of discretize.",
+            DeprecationWarning,
+        )
```

Comment:
Strangely the warning doesn't print on my end... even though I am sure I am getting there

![image](https://user-images.githubusercontent.com/55204635/204149169-3f1e124b-6649-4efe-859d-7b4e9bb44481.png)